e2e_test/nexmark/create_sources.slt.part

```diff
@@ -32,6 +32,40 @@ CREATE SOURCE nexmark (
     nexmark.min.event.gap.in.ns = '100'
 );
 
+statement error The shema of the nexmark source must specify all columns in order
```

Comment:
Seems not a good idea to put it into this file. 🥵 What about introducing a new `bug_fixes`?

Reply:
introduced a new file.